Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -309,6 +309,9 @@ class PlaceDetailViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
+    /// MAYBE DON'T USE THIS METHOD - may be the cause of #492. Opens the place card for the event
+    /// by inserting it into the linked list (which is a bit sketchy). You probably want
+    /// `openCard(forExistingPlace:)` instead.
```

Comment:
I figured we could remove the method entirely in a separate issue – iirc, the call sites are just for notifications.